kbase.yml

```diff
@@ -8,10 +8,10 @@ service-language:
     python
 
 module-version:
-    0.2.1
+    0.2.2
 
 owners:
-    [rsutormin, msneddon, wjriehl]
+    [rsutormin, msneddon, wjriehl, tgu2]
```

Comment:
You can remove rsutormin and msneddon at this point, I think.

Reply:
👍